packages/ember/src/index.ts

```diff
@@ -134,6 +135,12 @@ module.exports = {
     this._super.included.apply(this, args);
   },
 
+  // Re-enables caching of this addon, due to opting out
+  // of the caching implicitly by specifying treeFor* methods
+  cacheKeyForTree(treeType) {
+    return cacheKeyForTree(treeType, this);
+  },
```

Comment:
I'm pretty sure it's not a persistent (e.g. across runs of `ember s`) cache.

Still feels safer to accurately describe the things which can change the behaviour of the `treeFor` hook so that this cache doesn't become the thing which stops rebuilds from working in future...

Reply:
I like @vitch approach here. @NullVoxPopuli Would you update the PR to take that into consideration?

Reply:
ye